shallowblue_rllib.py

```diff
@@ -113,15 +126,15 @@ def step(self, action):
             self.agent_host.sendCommand('turn 0')
             self.agent_host.sendCommand('jump 0')
             self.agent_host.sendCommand('attack 1')
-            time.sleep(1.5)
-        elif (self.air_level < 0.3 * self.full_air) and action[2] > 0:
+            time.sleep(2)
```

Comment:
sometimes it fails to mine a block still? idk if we should just keep increasing sleep